Open external links in new tab (#403)
VertaAI/modeldb: client/verta/docs/support.rst

Support & Community
===================

Verta is an open-core MLOps platform and we provide a variety of ways for our users to
engage with our team, provide feedback, and get help!

* Check out our `Resources <learn.html>`_, `Tutorials &  Examples <examples.html>`_, and `FAQs <faqs.html>`_ sections for guides and answers to basic queestions.

> * Verta |GitHub| all have an Issues track for users to ask questions, create support tickets, and submit feature requests.

Author: ```suggestion
* Verta |Git repositories| all have an Issues track for users to ask questions, create support tickets, and submit feature requests.
```

> * Check out our |Slack| for open-source and basic support.

Author: ```suggestion
* Check out our |Verta Community Slack channel| for open-source and basic support.
```

* If you are a Verta Enterprise customer, check with your Verta admin or email `support@verta.ai` for your organization-specific Slack information.

> .. |GitHub| raw:: html

Author: ```suggestion
.. |Git repositories| raw:: html
```

   <a href="https://github.com/VertaAI/" target="_blank">Git repositories</a>

> .. |Slack| raw:: html

Author: ```suggestion
.. |Verta Community Slack channel| raw:: html
```
